### PR TITLE
Faq

### PR DESCRIPTION
**Pull request by AWare**
## Why are you doing this?

New FAQ list from support. To lessen load during and post inauguration. 

## FAQ
* Is this a big change
  * Not really

## Screenshots

image

nb paging @guardian/contributions there are some questions relating to contributions, so a tracking link could be a good idea

### CODE REVIEW

File: frontend/npm-debug.log.3061510990 [hunk 1]
```diff
@@ -0,0 +1,26 @@
+0 info it worked if it ends with ok
```
Comment (desbo) [inline frontend/npm-debug.log.3061510990]: 🙈

**Comment by desbo** [comment]
Looks good to me. I'd use `INTCMP=membership_faq`.

File: frontend/npm-debug.log.3061510990 (hunk 1 above)
Comment (AWare) [inline frontend/npm-debug.log.3061510990]: ugh
oops

File: frontend/assets/stylesheets/components/_faqs.scss [hunk 2]
```diff
@@ -4,9 +4,43 @@
...
 .faq {
     padding-bottom: $gs-gutter;
+    input[type=checkbox] {
+        display: none;
+    }
+    input[type=checkbox] ~ .faq__answer {
+        max-height: 0;
+        opacity: 0;
+        overflow: hidden;
+        transition: all 0.8s ease;
```
Comment (Ap0c) [inline frontend/assets/stylesheets/components/_faqs.scss]: The transition is nice, but it feels slightly slow at the moment to me 🏎. What do you think about dropping this to `0.1s`?

File: frontend/assets/stylesheets/components/_faqs.scss [hunk 3]
```diff
@@ -4,9 +4,43 @@
...
+        max-height: 100%;
+        opacity: 1;
+    }
 }
+
 .faq__headline {
     @include fs-headline(3);
-    font-weight: guss-font-weight(bold);
+    font-weight: guss-font-weight(medium);
     margin-bottom: $gs-baseline / 4;
```
Comment (Ap0c) [inline frontend/assets/stylesheets/components/_faqs.scss]: To make it slightly more obvious on desktop that these are click to dropdown, could we add `cursor: pointer`?

Comment (AWare) [inline frontend/assets/stylesheets/components/_faqs.scss]: done :)

Comment (Ap0c) [inline frontend/assets/stylesheets/components/_faqs.scss]: Looks good!

**Pull request merged**